projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,120 @@
+
+ 
+Additionally, the project allows us to bridge and advance the interest of both the OpenWallet Foundation and Mendix communities.  By collaborating with the Mendix community, we not only have the ability to grow the number of digital trust technology practitioners, we also tap into a community with an established customer base. By making digital wallets components that can be easily integrated into their existing solutions, the project can drive further adoption and usage of interoperable, secure and privacy preserving digital wallets. 
+
+ 
+
+## Origin & History 
+
+The origins of the **Farmworker Wallet OS** project align with those of Entidad. What began as a volunteering effort by three college friends, was formalized in 2018 with the founding of Entidad as a public benefit corporation.  We have primarily been working with leading farm worker-serving organizations to develop technology that leverages growing digital literacy in their communities to scale their impact.  
+
+ 
+The first digital service launched supported the UFW Foundation’s emergency relief efforts during the height of the COVID-19 pandemic. The solution was used to plan, manage and operate over 500 in-person community events where over $15 million in resources were distributed to over 40K families. Additionally, it has allowed us to better understand the unique challenges of building for underserved communities and why interoperable, secure, and privacy preserving technologies are key to addressing these challenges.  
+
+  
+You can read more about our journey on [our blog](https://www.entidad.io/blog). 
+
+# Alignment with the OpenWallet Foundation Mission
+The **Farmworker Wallet OS** project is a community of contribution led by Entidad and the United Farm Workers Foundation (UFWF) with the goal of furthering the adoption of an open, secure, interoperable digital wallet solution that makes it easier for farm worker communities to access life-changing social and human services. The open-source software components maintained by this community will enable non-profit and social good organizations to build wallet-driven solutions on top of the Mendix low-code platform. Collaboration with app makers from the broader Mendix developer community will help inform and guide the Farmworker Wallet OS project with industry proven best practices, leading to the implementation of reusable functionality that is aligned with the goals of the Open Wallet Foundation (to preserve user choice, security and privacy). The community will also inspire the next generation of low-code app builders to learn and expand their skill sets with decentralized (web3) technologies. 
+
+# Current Code of Conduct
+[Farmworker Wallet OS Community Code of Conduct](https://github.com/Entidad/.github/blob/main/CODE_OF_CONDUCT.md)
```

Comment:
It would be great to understand how this differs from the [OWF code of conduct](https://8112310.fs1.hubspotusercontent-na1.net/hubfs/8112310/Exhibit%20B%20--%20Linux%20Foundation%20Europe%20Policies.pdf). 

Question for the OpenWallet Foundation staff: Do we need to get this approved by LF Europe?

Reply:
Happy to use the OWF code of conduct. We didn’t realize that was an option. No need for a review with LF Europe. We will replace the existing code of conduct with the OWF’s.

Reply:
That would be great.

Reply:
Code of conduct has been updated to reference the OWF Code of Conduct